plugins/fabrik_list/inlineedit/inlineedit.js

```diff
@@ -170,40 +198,32 @@ var FbListInlineEdit = new Class({
 			}
 			if (typeOf(this.td.getPrevious()) === 'element') {
 				e.stop();
-				this.select(e, this.td.getPrevious());
+				this.select(e, this.getPreviousEditable(this.td));
 			}
 			break;
 		case 40:
 			//down
-			if (this.inedit) {
-				return;
+			if (this.inedit && this.options.tabSave) {
```

Comment:
I will get back to you on this - I'll  have to test that. But it's really not something that I added. The tab option, and the exception for the enter key in a textarea, was always there.